policyengine_canada/parameters/gov/provinces/nb/tax/income/credits/spouse_or_common_law_partner_amount/base_amount.yaml

```diff
@@ -0,0 +1,10 @@
+description: New Brunswick provides the following Spouse or Common-law partner base amount.  
+values:
+  2023-01-01: 10_741
+metadata:
+  unit: currency-CAD
+  period: year
+  label: New Brunswick Spouse or Common-law Partner Base Amount
+  reference:
+    - title: Worksheet for the 2023 New Brunswick Personal Tax Credits Return
+      href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1nb-ws/td1nb-ws-23e.pdf#page=1
```

Comment:
we will need to find the 2022 reference as well as the legal code for this program